en/20/01.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Chainlink Data Feeds Introduction
+actions: ['checkAnswer', 'hints']
+requireLogin: true
+material:
+  editor:
+    language: sol
+    startingCode:
+      "PriceConsumerV3.sol": |
+        // Start here
+    answer: |
+      pragma solidity ^0.6.7;
+
+      contract PriceConsumerV3 {
+
+      }
+---
+
+Now, let's suppose you're building a DeFi dapp, and want to give your users the ability to withdraw ETH worth a certain amount of USD. To fulfill this request, your smart contract (for simplicity's sake we'll call it the "caller contract" from here onwards) must know how much one Ether is worth.
+
+And here's the thing: a JavaScript application can easily fetch this kind of information, making requests to the Binance public API (or any other service that publicly provides a price feed). But, a smart contract can't directly access data from the outside world. 
+
+Now we could build a javascript application ourselves, but then we are introducing a centralized point of failure! We also couldn't just pull from the Binance API, because that would be a centralized point of failure!
```

Comment:
```suggestion
Now we could build a JavaScript application ourselves, but then we are introducing a centralized point of failure! We also couldn't just pull from the Binance API, because that would be a centralized point of failure!
```